blazor/datagrid/how-to/define-events-programatically.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Define the events programmatically in Blazor DataGrid Component | Syncfusion
+description: Checkout and learn here all about Define the events programmatically in Syncfusion Blazor DataGrid component and more.
```

Comment:
Checkout and learn here all about defining events programmatically in the Syncfusion Blazor DataGrid component and more.